docs/src/main/sphinx/connector/faker.md

```diff
@@ -0,0 +1,130 @@
+# Faker connector
+
+The Faker connector generates realistically looking random data matching a
```

Comment:
Somewhere in here link to datafaker.net

Reply:
See my other comment, using Datafaker in this connector is not the core feature.

Reply:
ok .. I think we probably still want to explain the fact that it is used in to the intro